packages/fast-usdc/test/exos/status-manager.test.ts

```diff
@@ -1,19 +1,19 @@
 import type { TestFn } from 'ava';
 
-import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
-import type { StorageNode } from '@agoric/internal/src/lib-chainStorage.js';
-import { eventLoopIteration } from '@agoric/internal/src/testing-utils.js';
 import { defaultMarshaller } from '@agoric/internal/src/storage-test-utils.js';
+import { eventLoopIteration } from '@agoric/internal/src/testing-utils.js';
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
```

Comment:
At least for consistency, should this `prepare-test-env...` import be moved to the top of the file?